watertap/unit_models/thickener.py

```diff
@@ -47,16 +49,49 @@
 _log = idaeslog.getLogger(__name__)
 
 
+class ActivatedSludgeModelType(Enum):
+    """
+    ASM1: ASM1 model
+    ASM2D: ASM2D model
+    modified_ASM2D: modified ASM2D model for ADM1 compatibility
```

Comment:
@adam-a-a Is this thickener only for use in BSM2 (as the description says)? If so, I don't think it makes sense to include regular ASM2d as a config option since it is incompatible with BSM2. It might still be worth keeping the changes you made to `asm2d_properties.py` though.

Reply:
I should change that file description because a user could be interested in assembling a flowsheet with a thickener that could make use of ASM2D without the need to convert to ADM1.